ddtrace/_trace/tracer.py

```diff
@@ -443,42 +441,127 @@ def configure(
 
         :param bool enabled: If True, finished traces will be submitted to the API, else they'll be dropped.
-        :param str hostname: Hostname running the Trace Agent
-        :param int port: Port of the Trace Agent
-        :param str uds_path: The Unix Domain Socket path of the agent.
-        :param bool https: Whether to use HTTPS or HTTP.
+        :param str hostname: Hostname running the Trace Agent. This parameter is deprecated and will be removed.
+        :param int port: Port of the Trace Agent. This parameter is deprecated and will be removed.
+        :param str uds_path: The Unix Domain Socket path of the agent. This parameter is deprecated and will be removed.
+        :param bool https: Whether to use HTTPS or HTTP. This parameter is deprecated and will be removed.
         :param object sampler: A custom Sampler instance, locally deciding to totally drop the trace or not.
+            This parameter is deprecated and will be removed.
         :param object context_provider: The ``ContextProvider`` that will be used to retrieve
             automatically the current call context. This is an advanced option that usually
             doesn't need to be changed from the default value
         :param object wrap_executor: callable that is used when a function is decorated with
             ``Tracer.wrap()``. This is an advanced option that usually doesn't need to be changed
             from the default value
-        :param priority_sampling: This argument is deprecated and will be removed in a future version.
+        :param priority_sampling: This parameter is deprecated and will be removed in a future version.
         :param str dogstatsd_url: URL for UDP or Unix socket connection to DogStatsD
+            This parameter is deprecated and will be removed.
+        :param TraceWriter writer: This parameter is deprecated and will be removed.
+        :param bool partial_flush_enabled: This parameter is deprecated and will be removed.
+        :param bool partial_flush_min_spans: This parameter is deprecated and will be removed.
+        :param str api_version: This parameter is deprecated and will be removed.
+        :param bool compute_stats_enabled: This parameter is deprecated and will be removed.
+        :param bool appsec_enabled: This parameter is deprecated and will be removed.
+        :param bool iast_enabled: This parameter is deprecated and will be removed.
+        :param bool appsec_standalone_enabled: This parameter is deprecated and will be removed.
+        :param List[TraceProcessor] trace_processors: This parameter sets TraceProcessor (ex: TraceFilters).
+           Trace processors are used to modify and filter traces based on certain criteria.
         """
+        if settings is not None:
+            deprecate(
+                "Passing settings to ``tracer.configure(...)`` is deprecated",
+                message="Please use the trace_proccessor parameter instead of settings['FILTERS'].",
```

Comment:
```suggestion
                message="Please use the trace_processor parameter instead of settings['FILTERS'].",
```

Reply:
Is settings only ever used for configuring trace filters?

Reply:
Yup, it was never expanded to anything else